kythe/rust/indexer/testdata/relative/relative_module/mod.rs

```diff
@@ -0,0 +1,14 @@
+//- RelativeModAnchor.node/kind anchor
+//- RelativeModAnchor.loc/start 0
+//- RelativeModAnchor.loc/end 0
+//- RelativeModAnchor defines/implicit RelativeMod
+//- RelativeMod.node/kind record
+//- RelativeMod.subkind module
+//- RelativeMod.complete definition
+
+//- @print_hello_world defines/binding FnPrint
+//- FnPrint.node/kind function
+//- FnPrint.complete definition
+pub fn print_hello_world() {
+    println!("Hello, world!");
+}
```

Comment:
nit: add a newline at the end of the file

Reply:
Thanks, I'm surprised pre-commit didn't catch this.